packages/type-safe-api/README.md

```diff
@@ -1035,6 +1041,146 @@ public class SayHelloHandler extends SayHello {
 }
 ```
 
+### Libraries
+
+Libraries are generated code projects which are not fully-fledged runtime languages.
+
+#### TypeScript React Query Hooks
+
+This library contains generated [react-query](https://tanstack.com/query/latest) hooks for interacting with your API from a React website. You can generate these by adding the following options to your `TypeSafeApiProject` in your `.projenrc`:
+
+```ts
+new TypeSafeApiProject({
+  library: {
+    libraries: [Library.TYPESCRIPT_REACT_QUERY_HOOKS],
+  },
+  ...
+});
+```
+
+##### Usage in a React Website
+
+First, make sure you create an instance of the API client (making sure to set the base URL and fetch instance). For example:
```

Comment:
NIT: Probably should call out that a dependency would be needed on the lib package.